pages/Fisherman-Androids.md

```diff
@@ -0,0 +1,39 @@
+A specialized android is created to automate the fishing process.
+
+## Obtaining
+Fisherman Androids can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+You have to put water underneath for the android to work.
+To power Androids you need compatible fuel source on each tiers.
```

Comment:
Androids need fuel in order to work, you have to fill them with the [Required Fuel](https://github.com/Slimefun/Slimefun4/wiki/Normal-Androids#power-source) before booting!